draft-ietf-tls-tls13.md

```diff
@@ -3073,10 +3073,15 @@ max_early_data_size
 
 The server is permitted to request client authentication at any time
```

Comment:
I'd probably also drop "at any time" since that kind of conflicts with the MUST NOT.